<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatteryTab</class>
 <widget class="QWidget" name="BatteryTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Cell Voltage ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Cell Voltage</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fanVoltage">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <double>12.330000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="fanSpeed">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>43</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>High Thermistor ID</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="averageTemp">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>52</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="bmsRelayStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>BMS Relay Status</string>
   </property>
   <widget class="QCheckBox" name="dischargeRelay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Discharge Relay</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chargeRelay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge Relay</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chargerSafety">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger Safety</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="malfunctionIndicator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Malfunction Indicator</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="alwaysOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Always On Signal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="multipurposeInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Multipurpose Input Signal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="isReady">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Ready Signal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="isCharging">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>145</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Charging Signal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="averageCellVoltage">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>190</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Temperature</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="packStateOfCharge">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <double>88.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>High Cell Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Requested Fan Speed</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="highCellVoltage">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>14</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>12v Input Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Internal Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pack Current</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="packAmphours">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <double>32.329999999999998</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="packVoltage">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <double>12.330000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="packCurrent">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <double>33.329999999999998</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pack Voltage</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="populatedCells">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>High Temperature</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="lowCellVoltageId">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="requestedFanSpeed">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>45</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="highCellVoltageId">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>High Cell Voltage ID</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="highTemp">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fan Speed</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="internalTemp">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>55</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="lowCellVoltage">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="lowTemp">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>44</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="lowThermistorId">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputVoltage12V">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <double>12.330000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pack Amphours</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fan Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Average Cell Voltage</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="packDepthOfDischarge">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <double>77.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pack State of Charge</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pack Depth of Discharge</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Populated Cells</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Thermistor ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Average Temperature</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="highThermistorId">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="alive">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alive</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
